File: java/171. Excel Sheet Column Number/Solution.java
class Solution {
    public int titleToNumber(String columnTitle) {

        int number = 0;

        for (int i = columnTitle.length() - 1; i >= 0; i--) {
            int digit = columnTitle.charAt(i) - 'A' + 1;
            number += digit * (int) Math.pow(26, columnTitle.length() - 1 - i);
        }

        return number;
    }
}

File: python/171. Excel Sheet Column Number/solution.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        number = 0
        for i in range(len(columnTitle) - 1, -1, -1):
            digit = ord(columnTitle[i]) - ord('A') + 1
            number += digit * int(math.pow(26, len(columnTitle) - 1 - i))

        return number

            